src/main/java/com/firebolt/jdbc/service/FireboltEngineInformationSchemaService.java

```diff
@@ -17,29 +16,38 @@
 import static java.lang.String.format;
 import static java.util.stream.Collectors.toCollection;
 
-@RequiredArgsConstructor
 public class FireboltEngineInformationSchemaService implements FireboltEngineService {
     private static final String ENGINE_URL = "url";
     private static final String STATUS_FIELD = "status";
     private static final String ENGINE_NAME_FIELD = "engine_name";
     private static final Collection<String> RUNNING_STATUSES = Stream.of("running", "ENGINE_STATE_RUNNING").collect(toCollection(() -> new TreeSet<>(CASE_INSENSITIVE_ORDER)));
     private static final String ENGINE_QUERY =
-            "SELECT engs.url, engs.attached_to, dbs.database_name, engs.status, engs.engine_name " +
+            "SELECT engs.url, engs.attached_to, dbs.%1$s_name, engs.status, engs.engine_name " +
             "FROM information_schema.engines as engs " +
-            "LEFT JOIN information_schema.databases as dbs ON engs.attached_to = dbs.database_name " +
+            "LEFT JOIN information_schema.%1$ss as dbs ON engs.attached_to = dbs.%1$s_name " +
             "WHERE engs.engine_name = ?";
-    private static final String DATABASE_QUERY = "SELECT database_name FROM information_schema.databases WHERE database_name=?";
+    private static final String DATABASE_QUERY = "SELECT %1$s_name FROM information_schema.%1$ss WHERE %1$s_name=?";
```

Comment:
Should this be called IS_QUERY or something? Since it's not only for querying databases

Reply:
I agree that now the name is not good enough, however IMHO `IS_QUERY` even worse. What about `EXISTENCE_QUERY` or, probably `INVENTORY_QUERY`?
But taking in consideration your next comment all this is irrelevant and we just have to  use catalogs instead of databases.

Reply:
We can use both suggested names. I'm just thinking to make the naming more general

Reply:
Renamed to `INVENTORY_QUERY`.